src/taskpane/components/TextInsertion.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { useState } from "react";
+import { Button, Field, Textarea, tokens, makeStyles } from "@fluentui/react-components";
+import { selectInsertionByHost } from "../../../test/end-to-end/src/host-relative-text-insertion";
```

Comment:
You cannot reference test code here. The test code can reference the add-in source code but not the other way around.

Reply:
The update convertToSingleHost.js replaces this line, so it's not in the source code of the emitted projects.